reframe/frontend/executors/policies.py

```diff
@@ -65,6 +65,8 @@ def run_check(self, check, partition, environ):
 
 
 class PollRateFunction:
+    _min_a = 0.01
```

Comment:
Does it make sense to have this variable defined in the `settings.py`?
Or Have it defined capitalized at the beginning of the file?

Reply:
@victorusu Generally, yes, but not in this issue. I will create another one.

Reply:
I added this one so that I do not enter a magic number in the middle of the code.

Reply:
@victorusu Another problem with exposing `_min_a` as-is is that it doesn't have a physical meaning. It is not the minimum poll rate.